locales/en/breaches.ftl

```diff
@@ -33,6 +33,20 @@ column-status-badge-resolved = Resolved
 # “Active” is shown next to a breach if the user still has at least one recommended action to perform in response to the breach.
 column-status-badge-active = Active
 
+breaches-none-headline = No breaches found
+# Variables:
+#   $email (String) - An email address that we did not find breaches for, e.g. `someone@example.com`
+breaches-none-copy = Good news! No known breaches were reported for { $email }. We’ll keep monitoring this email and will let you know if any new breaches occur.
+breaches-none-cta-blurb = Would you like to monitor another email?
+breaches-none-cta = Add email address
+
+breaches-all-resolved-headline = All breaches resolved
+# Variables:
+#   $email (String) - An email address for which all breaches have been resolved, e.g. `someone@example.com`
+breaches-all-resolved-copy = Nicely done! You’ve resolved all breaches for { $email }. We’ll keep monitoring this email and will let you know if any new breaches occur.
+breaches-all-resolved-cta-blurb = Would you like to monitor another email?
+breaches-all-resolved-cta = Add email address
```

Comment:
```suggestion
breaches-all-resolved-cta-button = Add email address
```

It's nice to know from the ID where the string is used. Same for `breaches-none-cta`.

Reply:
Heh, I actually went back and forth on that :sweat_smile: Changed in 731b7a05.